apps/src/gamelab/GameLab.js

```diff
@@ -244,6 +264,15 @@ GameLab.prototype.calculateVisualizationScale_ = function () {
 };
 
 /**
+ * @param {string} code The code to search for Data Storage APIs
+ * @return {boolean} True if the code uses any data storage APIs
+ */
+GameLab.prototype.hasDataStoreAPIs = function (code) {
+  return /createRecord/.test(code) || /updateRecord/.test(code) ||
+      /setKeyValue/.test(code);
+};
```

Comment:
Are we ever going to have data store APIs in GameLab? I would think these just won't exist there (but maybe I'm wrong)